AndroidSDKCore/src/main/java/com/leanplum/internal/VarCache.java

```diff
@@ -322,6 +323,14 @@ public static boolean hasReceivedDiffs() {
     return hasReceivedDiffs;
   }
 
+  public static Map<String, Object> getVariantDebugInfo() {
+    return variantDebugInfo;
+  }
+
+  public static void setVariantDebugInfo(Map<String, Object> _variantDebugInfo) {
+    variantDebugInfo= _variantDebugInfo;
```

Comment:
Auto format according to our code style, please.

Reply:
Same as above

Reply:
There is a space before `=`.  That is definitely not our style or standard Java code style.